src/pyscipopt/scip.pxi

```diff
@@ -5553,6 +5576,118 @@ cdef class Model:
         assert isinstance(var, Variable), "The given variable is not a pyvar, but %s" % var.__class__.__name__
         PY_SCIP_CALL(SCIPchgVarBranchPriority(self._scip, var.scip_var, priority))
 
+    def startStrongBranching(self):
+        """Start strong branching. Needs to be called before any strong branching. Must also later end strong branching.
+        If propagation is enabled then strong branching is not done on the LP, but on additionally created nodes (has some overhead)"""
+
+        PY_SCIP_CALL(SCIPstartStrongbranch(self._scip, False))
```

Comment:
Is there a reason for not allowing propagation? If there is, then I think the last sentence of the docstring doesn't make sense.

Reply:
I've added a TODO to the docstring. I originally did add the propagation option, but then decided to remove it. I did so because:
- One then needs to add a `bool` argument to `startStrongBranching` and `endStrongBranching` (if not set appropriately then the propagation option raises an error)
- There are many more return arguments of the function when `propagation=True`. This arguments are also useless unless other SCIP functions are then wrapped and made available. 
For these two reasons I just wanted to ignore it for now. I think the majority of users don't need this functionality, and if someone asks for it then we can add it. Or we let them add it themselves.

For the naming conventions: Any opinion on changing the name of the Python wrapper function @mmghannam ? I just couldn't stand the SCIP names for strong branching (found them incredibly unintuitive). I can change them back if needed.

Reply:
Sorry for the super late reply. I have thought a bit about this and I think I'm more in favour of keeping the same names as SCIP uses for the following reasons: 

1. SCIP's function names are not really likely to change in the future, so having the same names keeps a direct link with SCIP making it easy to transfer your knowledge to directly work in SCIP or easily use PySCIPOpt if you're already used to the C-API. 
2. Regarding the clarity issue, the new names could be clearer than the old ones but what would be more important in understanding what the function does is having a clear documentation, or maybe in this case a strong branching guide since the functionality spans multiple functions (and a link to it in the docstrings). 

So, I think changing the names of direct "wrapper" methods would do more harm than good.  However, I'm in favour of coming up with new names if the function does something more than a direct call to a SCIP function, and I think we should mark these functions somehow, maybe in the naming style; we could use snake case for these more elaborate methods.

Reply:
Summary: Agreed with Mo.